tuf/scripts/repo.py

```diff
@@ -1095,9 +1099,9 @@ def parse_arguments():
   parser.add_argument('-t', '--terminating', action='store_true',
       help='Set the terminating flag to True.  Can be used with --delegate.')
 
-  parser.add_argument('--threshold', type=int, default=1, metavar='<int>',
+  parser.add_argument('--threshold', type=int, metavar='<int>',
```

Comment:
Why no more default=1?

Reply:
since --threshold is now optionally given to --trust as well, it should not have a numeric default: otherwise using --trust without --threshold leads to threshold getting reset to 1 which is unexpected.

Reply:
Got it, so we should add this note as a comment above 🙂